sharktank/sharktank/layers/paged_attention.py

```diff
@@ -29,9 +29,98 @@
     StaticScaledQuantizer,
 )
 from sharktank import ops, kernels
+from sharktank.kernels.mlir_kernel import *
 
 __all__ = ["PagedAttention"]
 
+# Paged Attention Kernels
```

Comment:
This should be moved to sharktank/kernels/

Reply:
We can keep this inlined here. The attention gather fusion is painful enough that we will want dependencies to be largely centralized. There will also likely be some organizational redesign once Latent Attention is on point.